src/Meziantou.AspNetCore.Components/InfiniteScrolling/InfiniteScrolling.razor

```diff
@@ -66,6 +66,7 @@
         _loadItemsCts = cts;
         try
         {
+            _loading = true;
```

Comment:
In fact, the `_loading` field should not exist. It should have been replaced with `private bool IsLoading => _loadItemsCts != null;`. Do you want to do the change?

Reply:
Thanks for reporting the bug. It is fixed by 7a3278545db69d6a44062d67628610cb98a688a9.

Reply:
> In fact, the `_loading` field should not exist. It should have been replaced with `private bool IsLoading => _loadItemsCts != null;`. Do you want to do the change?
> 
> note: the bug was introduced in 5ca8395b87e71b2e6813f1ddb3ec69fc389fa994

This also what I had in mind, but was not sure you expect to do it like this. Perfect.